sites/public/layouts/forms.tsx

```diff
@@ -1,5 +1,6 @@
 import Layout from "./application"
 import { ApplicationTimeout } from "../src/forms/applications/ApplicationTimeout"
+import React from "react"
```

Comment:
Was this previously removed for some reason? Isn't that supposed to be on every react page/file?

Reply:
I'm not sure, but yeah without this import the test complains `'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.`